[2025-03-05T20:28:18.018+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-05T20:28:18.024+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: forecast_stock_prices.merge_forecast_into_final_table manual__2025-03-05T20:28:11.783908+00:00 [queued]>
[2025-03-05T20:28:18.028+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: forecast_stock_prices.merge_forecast_into_final_table manual__2025-03-05T20:28:11.783908+00:00 [queued]>
[2025-03-05T20:28:18.029+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-05T20:28:18.034+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): merge_forecast_into_final_table> on 2025-03-05 20:28:11.783908+00:00
[2025-03-05T20:28:18.038+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=927) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-05T20:28:18.039+0000] {standard_task_runner.py:72} INFO - Started process 947 to run task
[2025-03-05T20:28:18.039+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'forecast_stock_prices', 'merge_forecast_into_final_table', 'manual__2025-03-05T20:28:11.783908+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/forecast_stock_prices.py', '--cfg-path', '/tmp/tmp8y34h8_6']
[2025-03-05T20:28:18.040+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask merge_forecast_into_final_table
[2025-03-05T20:28:18.056+0000] {task_command.py:467} INFO - Running <TaskInstance: forecast_stock_prices.merge_forecast_into_final_table manual__2025-03-05T20:28:11.783908+00:00 [running]> on host 83a9dc73146e
[2025-03-05T20:28:18.084+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='forecast_stock_prices' AIRFLOW_CTX_TASK_ID='merge_forecast_into_final_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T20:28:11.783908+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-05T20:28:11.783908+00:00'
[2025-03-05T20:28:18.085+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-05T20:28:18.090+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-05T20:28:18.091+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-05T20:28:18.092+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-05T20:28:18.092+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-05T20:28:18.640+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-05T20:28:19.805+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-05T20:28:19.884+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-05T20:28:19.884+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-05T20:28:19.888+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-05T20:28:19.888+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=forecast_stock_prices, task_id=merge_forecast_into_final_table, run_id=manual__2025-03-05T20:28:11.783908+00:00, execution_date=20250305T202811, start_date=20250305T202818, end_date=20250305T202819
[2025-03-05T20:28:19.903+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-05T20:28:19.909+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-05T20:28:19.909+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
